### ISSUE TITLE
[shelly] Battery powered devices stay in CONFIG_PENDING rather than ONLINE

### ISSUE THREAD

**Issue by markus7017** [openhab/openhab-addons#15982]
Oncebattery powered device is initialized the thing changes to state ONLINE. From time to tie the binding tries to connect to the device and state is changed to CONFIG_PENDING, which is not correct. Such a thing should stay ONLINE unless communication failed or no updates are reported within the watchdog durations (e.g. 720min for H&T)

**Comment by alaub81** [comment]
I have a similar problem. My battery powered ShellyPlus HT Device is not able to update its state. After a bit of playing around, I found out, that the web socket URL is not available anymore. Last functional openHAB Version was 4.1.0.M3. I had the problem with all later Milestones and even now with the stable 4.1.0 Version.

I tried to open the web socket connection in a browser. With Milestone M3 I got an error, that the request is invalid. With the other versions, it is not available and hits the openhab Standard error Page.
